docs/scos/dev/back-end-development/extend-spryker/spryker-os-module-customisation/spryker-os-module-customisation.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Spryker OS module customisation
+description: This section lists documents inside the Spryker OS module customisation section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Spryker OS module customisation* section includes the following documents:
```

Comment:
Can you add a few words on what this section is about?

Reply:
```suggestion
Learn to customize modules in Spryker OS to meet your specific needs by extending core modules and exploring different approaches. Understand the folder structure used in Spryker Commerce OS and install module structure for project-level customization.

This section includes the following documents:
```